src/main/webapp/javascript/recipe-rendering.js

```diff
@@ -1,3 +1,19 @@
+// Set listeners for rendering based on user status,
+// except for User Page, which has special listeners.
+if (document.title != "My Recipes") {
+  // Listener that triggers when a new or different user signs in.
+  let userChanged = setIcons;
+
+  // Listener that triggers when sign-in status changes (but not when user changes).
+  let signInChanged = function(signedIn) {
```

Comment:
What is this meant to do besides console log? Do you need to include this code in the master branch?

Reply:
I inserted those listeners as placeholders for code I thought might be added by someone else. In this case, I expected that we would want to detect signout and reset all the icons to default.

Reply:
oh okay got it yeah I can do that